spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -20,110 +20,126 @@ On Ethereum, there are two types of accounts: externally owned accounts, control
 
-interface IBCAccountPacketData {
-  type: Type
-  data: Uint8Array
+`authenticateTx` validates the transaction and checks that the signers in the transaction have the right permissions (are the owners of the account in question).`
+
+`runTx` executes a transaction after the transaction has been successfully authenticated.
+
+### Packet Data
+`InterchainAccountPacketData` contains an array of messages that an interchain account can run and a memo string that is sent to the receiving chain. The example below is defined as a proto encoded message but each chain can encode this differently. This should be clearly defined in the documentation for each chain specific implementation.
+
+```typescript
+message InterchainAccountPacketData  {
+    repeated google.protobuf.Any messages = 1;
+    string memo = 2;
 }
 ```
 
-The acknowledgement data type describes the type of packet data and chain id and whether the result code of processing and the data of result if it is needed, and the reason for failure (if any).
+The acknowledgement packet structure is defined as in [ics4](https://github.com/cosmos/cosmos-sdk/blob/v0.42.4/proto/ibc/core/channel/v1/channel.proto#L134-L147). The acknowledgement result should contain the chain-id and the address of the targetted interchain account. If an error occurs on the receiving chain the acknowledgement should contain the error message.
 
 ```typescript
-interface IBCAccountPacketAcknowledgement {
-    type: Type
-    code: uint32
-    data: Uint8Array
-    error: string
+message Acknowledgement {
+  // response contains either a result or an error and must be non-empty
+  oneof response {
+    bytes  result = 21;
+    string error  = 22;
+  }
 }
 ```
 
-The ```IBCAccountHook``` interface allows the source chain to receive results of executing transactions on an interchain account.
-
+The ```InterchainAccountHook``` interface allows the source chain to receive results of executing transactions on an interchain account.
 ```typescript
-interface IBCAccountHook {
-  onAccountCreated(sourcePort:string, sourceChannel:string, address: Address)
+interface InterchainAccountHook {
   onTxSucceeded(sourcePort:string, sourceChannel:string, txBytes: Uint8Array)
   onTxFailed(sourcePort:string, sourceChannel:string, txBytes: Uint8Array)
 }
 ```
 
-### Subprotocols
-
-The subprotocols described herein should be implemented in an "interchain-account-bridge" module with access to a router and codec (decoder or unmarshaller) for the application and access to the IBC relayer module.
-
 ### Port & channel setup
+Receiving chains (chains that will host interchain accounts) must always bind to a port with the id `interchain_account`. Sending chains will bind to ports dynamically, with each port id being the address of the interchain account owner.
```

Comment:
Can a single account owner (identified with its address) be owning several interchain accounts on a given receiving chain?

Reply:
I'm also curious if a single account can own several interchain accounts on different receiving chains. I'd like to use a single account on chainA to create an account on chainB, chainC, and chainD and I can use my one authentication setup to control all 3 accounts